lti_consumer/api.py

```diff
@@ -34,6 +34,10 @@ def _get_or_create_local_lti_config(lti_version, block_location,
     the XBlock to be the source of truth for the LTI version, which is a user-centric perspective we've adopted.
     This allows XBlock users to update the LTI version without needing to update the database.
     """
+    # Replace CCX block locator with block locator to allow CCX launch to work.
```

Comment:
CCX Launch works if tool is configured properly. Could you please explain why now it works in terms of our context. I understand it does because the tool is only configured for the Master Course, otherwise configuring it for every new CCX only does not scale, that correct?
```suggestion
    # Replace CCX block locator with Master Course block locator...
```

Reply:
@Squirrel18 It now works because there is no LtiConfiguration being created for the CCX XBlock, each new LtiConfiguration for LTI 1.3 has a different client_id, which is not recognized by the tool provider, since CCXs are always a copy of the master course, we can make all LTI consumer XBlocks on the CCX course by using the master course counterpart of that block location.